### PR TITLE
Ui streaming data update

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher let me know what you want to do about this PR. and if you want me to push it up to the server.

### CODE REVIEW

File: ooiui/static/js/views/science/HighChartsStreamingDataView.js [hunk 1]
```diff
@@ -440,20 +452,29 @@ var HighchartsStreamingDataView = Backbone.View.extend({
...
+                    point = [dx._i,y]
 
                     if (i < points['data'].length-1){
                         self.chart.series[vv].addPoint(point, false, shift);
                     }else{
                         self.chart.series[vv].addPoint(point, true, shift);
                     }
+                  }
+                  //fix start date
+                  self.ds.startdate = moment.utc(x).format("YYYY-MM-DDTHH:mm:ss.000")+"Z"
```
Comment (birdage) [inline ooiui/static/js/views/science/HighChartsStreamingDataView.js]: @DanielJMaher use the last point as the new start date

File: ooiui/static/js/views/science/HighChartsStreamingDataView.js [hunk 2]
```diff
@@ -326,9 +327,9 @@ var HighchartsStreamingDataView = Backbone.View.extend({
 
     _.bindAll(this, "onClick",'requestData','abort','updateDateTimes','getUrl','updateVariable','resume');
 
-    var dt = moment();
-    var dt2Str = dt.format("YYYY-MM-DDTHH:mm:ss.000")+"Z"
-    var dt1Str = dt.subtract(10, 'seconds').format("YYYY-MM-DDTHH:mm:ss.000")+"Z"
+    var dt = moment().utc();
+    var dt2Str = self.getEndDate(dt,0);
+    var dt1Str = self.getStartDate(dt,300);
```
Comment (birdage) [inline ooiui/static/js/views/science/HighChartsStreamingDataView.js]: @DanielJMaher 5 min back check

File: ooiui/static/js/views/science/HighChartsStreamingDataView.js [hunk 3]
```diff
@@ -371,6 +383,7 @@ var HighchartsStreamingDataView = Backbone.View.extend({
   resume:function(){
     //kill the request
     this.multiRequest = true;
+    this.overrideStDate = true;
```
Comment (birdage) [inline ooiui/static/js/views/science/HighChartsStreamingDataView.js]: @DanielJMaher on resume get the last 5 minutes

**Comment by oceanzus** [comment]
Working on test server, merging.

**Pull request merged**